kubernetes/pre_post_install_task_generator.py

```diff
@@ -1,32 +1,26 @@
 import re
```

Comment:
Remove this is not used.

Reply:
it is used in separating hooks from manifests.